src/code/code_8012EC80.c

```diff
@@ -725,3 +725,591 @@ void Inventory_SaveLotteryCodeGuess(PlayState* play) {
     lotteryCodeGuess |= (play->msgCtx.unk12054[2] & 0xF);        // Third Digit
     HS_SET_LOTTERY_CODE_GUESS(lotteryCodeGuess);
 }
+
+u8 gPlayerFormItemRestrictions[PLAYER_FORM_MAX][114] = {
```

Comment:
Would be nice to have a define/enum for the 114, but can't really think on anything.
Anyways, thanks for formatting this array